packages/unmock-node/src/backend/index.ts

```diff
@@ -75,11 +75,16 @@ export default class NodeBackend implements IBackend {
   } = {};
   private origOnSocket?: (socket: net.Socket) => void;
   private mitm: any;
+  private stateStore: any;
```

Comment:
How come this has no type?

Reply:
`Proxy` returns type `any` AFAICS.

Reply:
Ok, did not realize that the state store is the proxy...

Reply:
Would renaming this be beneficial (`stateStoreProxy`)?

Reply:
Hmm not sure, maybe I just wasn't expecting a "store" to be the proxy handler for user-facing commands. But I don't have any suggestion for a better name.